server/snapshot/snapshot.go

```diff
@@ -1,11 +1,11 @@
 package snapshot
 
 import (
-	"encoding/json"
-
 	"github.com/Sirupsen/logrus"
 
+	"github.com/docker/go/canonical/json"
```

Comment:
Good point!  I've now added changes to the backwards compatibility test to also rotate a snapshot, which tests that hte new snapshot gets generated ok with old data.  (https://github.com/docker/notary/pull/568/files#diff-7b61277673385a0a9b278b5bec4c9ffaR131)

Do you have any other suggestions for tests?